showcase-docs/dynamic-plugins.md

```diff
@@ -44,12 +41,12 @@ These recommended changes to the `package.json` are summarized below:
   ...
   "dependencies": {
     ...
-    "@backstage/backend-dynamic-feature-service": "0.1.0",
+    "@backstage/backend-dynamic-feature-service": "0.2.3",
```

Comment:
In fact that should not even be necessary in most cases, when using a plugin made for the new backend system.

Reply:
So line https://github.com/janus-idp/backstage-showcase/pull/1105/files#diff-8d2e5712d5ef532a983602f4c771597e41d8c082d7801f59f6003fe9f66e2a89L28 can be removed.

Line https://github.com/janus-idp/backstage-showcase/pull/1105/files#diff-8d2e5712d5ef532a983602f4c771597e41d8c082d7801f59f6003fe9f66e2a89R28 should also be removed

Reply:
In line 38 you should use:

```
          "export-dynamic": "janus-cli package export-dynamic-plugin ----embed-as-dependencies"
```